doc/topics/jinja/index.rst

```diff
@@ -1700,6 +1700,23 @@ Will insert the following message in the minion logs:
 
 .. jinja_ref:: custom-execution-modules
 
+Python methods
```

Comment:
I had considered this, but found that not all titles were completely capitalized.